app/views/devise/registrations/edit.html.haml

```diff
@@ -6,9 +6,17 @@
   = form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
     = devise_error_messages!
```

Comment:
I've made the requested change, with one minor difference: I had to add a break after the individual messages. The code above as is would put multiple error messages on one line.